scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/scheduler/SchedulerHandler.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.scheduling.scheduler;
+
+import com.amazonaws.blox.dataservicemodel.v1.client.DataService;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentRevision;
+import com.amazonaws.blox.dataservicemodel.v1.model.operations.DescribeEnvironmentRevisionResponse;
+import com.amazonaws.blox.dataservicemodel.v1.model.operations.DescribeTargetEnvironmentRevisionRequest;
+import com.amazonaws.blox.scheduling.scheduler.engine.Scheduler;
+import com.amazonaws.blox.scheduling.scheduler.engine.SchedulerFactory;
+import com.amazonaws.blox.scheduling.scheduler.engine.SchedulingAction;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.spotify.futures.CompletableFutures;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.services.ecs.ECSAsyncClient;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__({@Autowired}))
+@Log4j2
+public class SchedulerHandler implements RequestHandler<SchedulerInput, SchedulerOutput> {
+  private final DataService data;
+  private final ECSAsyncClient ecs;
+  private final SchedulerFactory schedulerFactory;
+
+  @Override
+  public SchedulerOutput handleRequest(SchedulerInput input, Context context) {
+    log.debug("Request: {}", input);
```

Comment:
delete?

Reply:
Will do once we have that logging handler (added a task for it to the issue).